src/main/java/uk/gov/companieshouse/api/accounts/controller/CurrentPeriodController.java

```diff
@@ -66,23 +63,23 @@ public ResponseEntity create(@RequestBody @Valid CurrentPeriod currentPeriod,
 
         }
 
+        Map<String, String> pathVariables = (Map) request
```

Comment:
if we can get this from the URI by adding a parameter to this controller method that would be nicer than using path variables